File: problemas/iniciante/1019/1019.cpp
#include <iostream>

using namespace std;

int main()
{
    int segundos, horas, minutos;

    cin >> segundos;

    horas = segundos / 3600;
    segundos %= 3600;
    minutos = segundos / 60;
    segundos %= 60;

    cout << horas << ':' << minutos << ':' << segundos << endl;

    return 0;
}

File: problemas/iniciante/1019/1019.go
package main

import (
    "fmt"
)

func main() {
    var segundos,minutos,horas int

    fmt.Scanf("%d", &segundos)

    horas = segundos / 3600
    segundos %= 3600
    minutos = segundos / 60
    segundos %= 60  

    fmt.Printf("%d:%d:%d\n", horas, minutos, segundos)
}
